README.md

```diff
@@ -80,6 +80,8 @@ The attributes listed below are used in *config.json* to configure **Spoor**, an
 
 >>**\_commitOnStatusChange** (boolean): Determines whether a "commit" call should be made automatically every time the SCORM *lesson_status* is changed. The default is `true`.  
 
+>>**\_commitOnAnyChange** (boolean): Determines whether a "commit" call should be made automatically every time any SCORM value is changed. The default is `false`. Note that setting **\_commitOnAnyChange** to `true` may have a large impact on the server and should only be used if absolutely necessary. An alternative is to use a lower **\_timedCommitFrequency** value as it will have a lower to impact on the server.
```

Comment:
```suggestion
>>**\_commitOnAnyChange** (boolean): Determines whether a "commit" call should be made automatically _every time_ any SCORM value is changed. The default is `false`. Setting **\_commitOnAnyChange** to `true` will disable 'timed commits'. **Note:** enabling this setting will make the course generate a lot more client-server traffic so you should only enable it if you are sure it is needed and, as it may have a detrimental impact on server performance, after careful load-testing. An alternative might be to first try setting a lower value for **\_timedCommitFrequency**.
```